pkg/kwokctl/runtime/binary/cluster.go

```diff
@@ -565,7 +565,89 @@ func (c *Cluster) Down(ctx context.Context) error {
 	return nil
 }
 
-func (c *Cluster) Start(ctx context.Context, name string) error {
+func (c *Cluster) Start(ctx context.Context) error {
+	conf, err := c.Config()
+	if err != nil {
+		return err
+	}
+
+	err = c.Up(ctx)
+	if err != nil {
+		return err
+	}
+
+	err = c.WaitReady(ctx, 30*time.Second)
+	if err != nil {
+		return fmt.Errorf("failed wait for cluster %q be ready: %w", conf.Name, err)
+	}
+
+	return nil
+}
+
+func (c *Cluster) Stop(ctx context.Context) error {
```

Comment:
What's the difference with Down?

Reply:
Stop() without unset kubeconfig

Reply:
Maybe we should separate the set/unset kubeconfig

Reply:
Is it OK for a separate pr to do this?